src/modules/RA/Scene/components/GeolayerSelect.js

```diff
@@ -69,8 +69,8 @@ const GeolayerSelect = ({ dataProvider, onChange, excludeIds = [], includeIds =
 
   const handleChoice = ({ target: { value } }) =>
     onChange({
-      geolayer: value,
-      label: geolayers.find(({ id }) => (value === id)).name,
+      geolayer: +value,
```

Comment:
I personally prefer the explicit `Number(value)` but it's the same

Reply:
Done.